testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/FlowOverrideTest.java

```diff
@@ -531,6 +561,52 @@ public void testClientOverrideFlowUsingBrowserHttpChallenge() {
         events.clear();
     }
 
+    // TODO remove this once DYNAMIC_SCOPES feature is enabled by default
+    @Test
+    @EnableFeature(value = Profile.Feature.DYNAMIC_SCOPES, skipRestart = true)
+    public void testClientOverrideFlowUsingBrowserHttpChallengeWithDynamicScope() {
+        // Just use existing test with DYNAMIC_SCOPES feature enabled as it was failing with DYNAMIC_SCOPES
+        Client httpClient = AdminClientUtil.createResteasyClient();
```

Comment:
Same comment as above. It seems it is possibly to just invoke `testClientOverrideFlowUsingBrowserHttpChallenge()` method from here directly?